server/src/Controller/Message/MessageController.js

```diff
@@ -0,0 +1,103 @@
+import {
+  create,
+  deleteDoc,
+  findOne,
+  findMany,
+  update,
+} from '../../Repositories/genericRepository';
+import successHandler from '../../libs/routes/successHandler';
+import { messageModel } from '../../Model';
+
+
+const getMessage = async (req, res, next) => {
+  try {
+    const { id } = req.params;
+    const result = await findOne(messageModel, id);
```

Comment:
What is the use-case of this API? Messages will always go through sockets

Reply:
Won't you need this for UI purpose when you would have to show threads?